pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -461,6 +462,16 @@ message CommandGetLastMessageIdResponse {
 	required uint64 request_id  = 2;
 }
 
+message CommandGetTopicsOfNamespace {
+	required uint64 request_id	= 1;
+	required string namespace	= 2;
+}
+
+message CommandGetTopicsOfNamespaceResponse {
+	required uint64 request_id	= 1;
+	repeated string topics 		= 2;
```

Comment:
It's probably not worth it or even necessary to bother about, but I was thinking how we could compress a little be this response. 

For one, we at least know that all topics have the same namespace (though they might have persistent and non-persistent topics together).

Reply:
Thanks. How about transfer back only the last part of local_topics_name?
If this is really a issue, maybe we could do the filter in server side, and only transfer back matched topics?